deploy/rp-production-subscription.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "resources": [
+        {
+            "properties": {
+                "groupShortName": "rphealth",
+                "enabled": true
+            },
+            "name": "rphealth-ag",
+            "type": "Microsoft.Insights/ActionGroups",
```

Comment:
lowercase a on ActionGroups throughout for consistency with the rest of the codebase